packages/core/src/indexing/database-index-api.ts

```diff
@@ -99,15 +99,15 @@ export class DatabaseIndexApi<DateType = Date | number> {
         models.map((indexModelArgs) => {
           return {
             model: indexModelArgs.model.toString(),
-            updated_by: '<FIXME: PUT ADMIN DID WHEN AUTH IS IMPLEMENTED>',
+            updated_by: '0', // TODO: FIXME: CDB-1866 - <FIXME: PUT ADMIN DID WHEN AUTH IS IMPLEMENTED>',
```

Comment:
curious: how is this used now?

Reply:
Currently not being used - was part of the initial simple audit trail approach to log recent activities before adding a more complete one thats referenced in CDB-1872.